app/forms/remote_search_form.rb

```diff
@@ -6,6 +6,8 @@ class RemoteSearchForm
     :checkbox,
     *SearchForm::TYPES_OF_ADVICE
```

Comment:
It feels inherently wrong to me that we're referencing one form from the other. But I guess we'll get through this when we bring these two things together anyway. So not specifically an action now, just idle commentary :smile: